app/models/piece.rb

```diff
@@ -18,12 +17,47 @@ def move!(x, y)
     true
   end
 
+  # Return false if coordinates is taken but belongs to friendly piece
+  # otherwise return true
+  def capturable?(x, y)
+    return false unless position_taken?(x, y) && !friendly_piece?(x, y)
+    true
+  end
+
+  # if piece is capturable change piece's attributes
+  def capture!(x, y)
+    return false unless capturable?(x, y)
+    move!(x, y)
+    target_piece(x, y).update_attributes!(
+      x_position: nil,
+      y_position: nil,
+      captured: true
+    )
+  end
```

Comment:
Worst case scenario just leave as is -- I'm going to have to refactor a chunk of the code anyway when I slot the big beast -- obstruction logic -- into it.  So I'm fine with not resolving this now.  Move, valid move, obstruction, and capture are going to merged anyway in my PR so let's not get hung up on this right now.